src/tools/compiletest/src/runtest.rs

```diff
@@ -1725,6 +1725,28 @@ impl<'test> TestCx<'test> {
         }
     }
 
+    fn is_vxworks_pure_static(&self) -> bool {
+        if self.config.target.contains("vxworks") {
+            match env::var("RUST_TEST_DYLINK") {
```

Comment:
Since this vxworks-specific could VXWORKS be included in the env var name somehow?

Reply:
> Since this vxworks-specific could VXWORKS be included in the env var name somehow?

Sure, I will change it to "RUST_VXWORKS_TEST_DYLINK".